_assets/js/expand-sidenav.js

```diff
@@ -0,0 +1,32 @@
+const sidenav = () => {
+    const expandButton = document.getElementById('expand-all');
+
+    function expandAll(e) {
+        const submenuButtons = document.getElementsByClassName('usa-accordion__button');
+        if (e.target.innerText === 'Expand all') {
+            Array.from(submenuButtons).forEach(button => {
+                const isExpanded = button.getAttribute('aria-expanded') === "true";
+                if (!isExpanded) {
+                    button.click();
```

Comment:
Because we're simulating a click of each button, does this cause lag on less performant devices?

Reply:
I've only been able to test on my laptop but it happens instantaneously so I think it should be fine?